test/e2e/scenarios/scenario1.yaml

```diff
@@ -0,0 +1,40 @@
+id: "RG_CMD_basic_create_update"
+description: "In this test we create RG with absolutely minimum configuration and try to update it."
+usecases:
+  - replication_group
+  - cmd
+#marks:
+#  - slow
+#  - blocked
```

Comment:
What's the difference between usecases and marks?

Reply:
I added some details related to it at: https://github.com/aws-controllers-k8s/elasticache-controller/pull/40#issuecomment-881827614 

I think currently marks are used to include tests that are by default excluded otherwise.
The intent of usecases was to have 'selector'/'filter' behavior such that when it is specified to pytest run command,   only tests that match the supplied usecase are included in the test run.

Note: the behavior is not yet implemented.